src/Config/CliBuilder.ts

```diff
@@ -64,13 +64,23 @@ export class CliBuilder {
       })
       .option('proxy', {
         requiresArg: true,
+        deprecated: true,
         describe: 'SOCKS4 or SOCKS5 URL to proxy all traffic'
       })
+      .option('proxy-external', {
+        requiresArg: true,
+        describe: 'SOCKS4 or SOCKS5 URL to proxy external traffic'
+      })
+      .option('proxy-internal', {
+        requiresArg: true,
+        describe: 'SOCKS4 or SOCKS5 URL to proxy internal traffic'
+      })
       .middleware((args: Arguments) => {
         ({ bus: args.bus, api: args.api } = Helpers.getClusterUrls(
           args as ClusterArgs
         ));
       }, true)
+      // TODO: (victor.polyakov@neuralegion.com) Write correct type checking
```

Comment:
What do you mean?

Reply:
It likes to `+(args.timeout as number),`
Using `as type` is a bad idea.